src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FeeStep.tsx

```diff
@@ -64,23 +43,23 @@ export const FeeStep = ({ isLastStep }: FeeStepProps) => {
   }, [])
 
   const { title, titleProps } = useMemo(() => {
-    return shapeShiftFee && shapeShiftFee.amountAfterDiscountUserCurrency !== '0'
-      ? { title: toFiat(shapeShiftFee.amountAfterDiscountUserCurrency) }
+    return amountAfterDiscountUserCurrency !== '0'
```

Comment:
That was my mistake to start with but `.gt(0)` would be better IMO - we should never have negative values, but you never know

Reply:
actioned in 19be7d23c47677edafae86e2457a7bc278d0e430